content/articles/what-are-ds-records.md

```diff
@@ -0,0 +1,138 @@
+- Below that are your domains, like `example.com`, referred to as child zones.
+
+In a DNSSEC-enabled setup, each layer can validate the one below it. The DS record allows the parent zone to securely delegate trust to your domain by linking to your domain's DNSSEC keys.
+
+## Why DS records matter
+
+When DNSSEC is enabled for your domain, DNS resolvers can verify that your DNS records haven't been forged or modified. This is possible, because your records are cryptographically signed using private keys, and the corresponding public key (your KSK) is published in your domain via DNSKEY records.
+
+But publishing a DNSKEY alone isn't enough. For DNS resolvers to trust your DNSKEY, they need a verified reference to it from the parent zone. That's the role of the DS record.
+
+Without a DS record in the parent zone, or if the DS record doesn't match your current KSK, DNS resolvers will reject your domain's DNSSEC signatures. This can make your domain unreachable to users using DNSSEC-validating resolvers.
+
+## How DS records establish trust
+
+Enabling DNSSEC involves a series of steps that lead to the creation and publication of a DS record. 
+
+Here's how it works:
+
+1. **Generate DNSSEC keys**
+    When you enable DNSSEC on your domain, two key pairs are created:
+   
+   - **Key Signing Key (KSK):** Used to sign the DNSKEY record set
+   - **Zone Signing Key (ZSK):** Used to sign your actual DNS records (like A, MX, etc.)
+
+    Both keys are published as DNSKEY records, but the KSK is the one trusted by the parent zone.
+
+1. **Create the DS record**
+    A cryptographic digest (hash) is generated from your domain's KSK. This becomes the DS record. It includes details about the algorithm and digest type used, but it does not expose your full key.
+
+1. **Publish the DS record in the parent zone**
+    The DS record is submitted to your domain registrar who publishes it through the domain registry (like .com or .org).
+
+    Once the DS record is live in the parent zone, it links your domain to the rest of the DNSSEC trust chain.
+
+1. **Resolver validates the Chain of Trust**
+    When a DNSSEC-aware resolver queries your domain, it:
+    - Retrieves the DS record from the parent zone
+    - Fetches the DNSKEY from your domain zone
+    - Compares the DS digest to your DNSKEY
+
+The resolver creates a digest of the DNSKEY using the same hash algorithm and digest type specified in the DS record. Even though the DS record doesn't contain the full public key, it includes a cryptographic hash of it. By hashing the DNSKEY locally and comparing it to the digest in the DS record, the resolver can confirm the two are linked.
+
+If the created digest matches the DS record, the resolver trusts your DNSKEY and the DNS records it signs. If they don't match — for example, if you've rotated your KSK but haven't updated the DS record — resolvers won't trust your domain's DNSSEC signatures, and your domain may become unreachable.
```

Comment:
> for example, if you've rotated your KSK but haven't updated the DS record

This is not a good example because the key rotation would create a new set of DS/DNSKEY records with a different key tag (resolvers will try to verify DS&DNSKEY records that match the same key tag).

A possible scenario for this would be a copy&paste error while provisioning the DS record in the parent zone, or malicious third-party tampering with DNS answers for the child zone.

Reply:
Sorry I didn't catch it earlier! 😅 I'll drop a suggestion in the doc